portal-frontend/src/app/features/edit-submission/primary-contact/primary-contact.component.ts

```diff
@@ -135,10 +135,7 @@ export class PrimaryContactComponent extends FilesStepComponent implements OnIni
         this.isCrownOwner = selectedOwner.type.code === APPLICATION_OWNER.CROWN;
       }
     }
-
-    this.needsAuthorizationLetter = !(
```

Comment:
What does this change do?

Reply:
I think this might not be enough, as now you are showing an error if they don't have a selected Owner, but what if they have a selected owner and its individual?

Reply:
Oh it's definitely not, sorry I'm still fixing it right now, I couldn't find the draft mode

Reply:
I noticed it created different bugs after committing the fix. I'll post on chat once it's ready for review!